providers_test.go

```diff
@@ -257,8 +254,5 @@ func TestGeneral_Syncing(t *testing.T) {
 		if !ok || i.Cmp(big.NewInt(0)) <= 0 {
 			t.Fatal("CurrentBlockNum should be positive number, instead: ", syncv02.CurrentBlockNum)
 		}
```

Comment:
Why was the prefix test moved? 

if !strings.HasPrefix(syncv02.CurrentBlockHash, "0x")..